helpers/role.js

```diff
@@ -0,0 +1,18 @@
+const { HttpCode } = require('../configuration/const');
+const { FORBIDDEN } = require('../messages/role-message');
+
+const guard = role => (req, res, next) => {
```

Comment:
Trochę tego nie rozumiem.

Reply:
Nie wiem jak się to ma do zadania, więć może nie wnikam :) Ale staraj się mieć nazwy bardziej opisowe, słowo guard mówi tylko tyle że coś chroni. I teraz trzeba przeglądać kod zeby zrozumieć. 
Więc bardziej opisowy język by się przydał, oraz komentarz na górze też może pomóc.